component-library/component-lib/src/lib/form-components/input/input.component.ts

```diff
@@ -365,6 +375,7 @@ export class InputComponent
   public clearvalue() {
     this.buttonAutoCompleteClearClicked = true;
     this.config.formGroup.controls[this.config.id].setValue('');
+    this.renderer.selectRootElement(this.inputEl?.nativeElement).focus();
```

Comment:
Turns out there is no better option than to manually trigger input field focus state when pressing clear button.